code/modules/events/spacevine.dm

```diff
@@ -561,8 +564,6 @@
 				master.spawn_spacevine_piece(stepturf, src)
 
 /obj/structure/spacevine/ex_act(severity, target)
-	if(istype(target, type)) //if its agressive spread vine dont do anything
-		return
```

Comment:
Is this meant to be removed?

Reply:
Yup. Its part of a pr jupyterkat ported half of.

Reply:
Huh